### ISSUE TITLE
Touchpad not working properly after update to Plasma 6

### ISSUE THREAD

**Issue by TraceyC77** [getsolus/packages#1911]
### Please confirm there isn't an existing open bug report

- [X] I have searched open bugs for this issue

### Summary

After updating to Plasma 6, the touchpad is not fully working

### Steps to reproduce

1. Go to Settings - Mouse & Touchpad - Touchpad

### Expected result

Settings for the touchpad are shown

### Actual result

The only thing on the pane is the text

"No touchpad found"

### Environment

- [X] Is system up to date?

### Repo

Unstable

### Desktop Environment

Plasma

### System details

Operating System: Solus 4.5
KDE Plasma Version: 6.0.2
KDE Frameworks Version: 6.0.0
Qt Version: 6.6.2
Kernel Version: 6.6.21-280.current (64-bit)
Graphics Platform: X11
Processors: 16 × 11th Gen Intel® Core™ i7-11800H @ 2.30GHz
Memory: 62.5 GiB of RAM
Graphics Processor: NVIDIA GeForce RTX 3060 Laptop GPU/PCIe/SSE2
Manufacturer: Dell Inc.
Product Name: XPS 17 9710

### Other comments

Synaptics driver is installed

```bash
❯ eopkg info xorg-driver-input-synaptics
Installed package:
Name                : xorg-driver-input-synaptics, version: 1.9.2, release: 32
```

The touchpad is working, but not all functionality is working.These are not working:
- Tap to click
- Two finger tap for right click
- Three finger tap for middle click

**Comment by Justinzobel** [comment]
If you run this from a terminal does it show that it's not found?
Mine for example with Plasma 6 and touchpad (my settings shows all the tickboxes etc)
```
kcmshell6 kcm_touchpad
kcm_touchpad: Touchpad found: "SynPS/2 Synaptics TouchPad" ("event6")
```

**Comment by TraceyC77** [comment]
Here are my results:

## Dell XPS - where problem reported

```bash
❯ kcmshell6 kcm_touchpad
qt.qml.typeresolution.cycle: Cyclic dependency detected between "qrc:/qt/qml/org/kde/desktop/private/TextFieldContextMenu.qml" and "qrc:/qt/qml/org/kde/desktop/MenuItem.qml"
qrc:/libinput/touchpad.qml:691: TypeError: Cannot read property 'supportsScrollFactor' of null
```

This brings up the touchpad settings pane, which says "No touchpad found"

## Lenovo Flex - does not have the problem

On the Flex, the touchpad works as expected.

```bash
$ kcmshell6 kcm_touchpad
kcm_touchpad: Touchpad found: "ELN4690:00 04F3:304B Touchpad" ("event7")
qt.qml.typeresolution.cycle: Cyclic dependency detected between "qrc:/qt/qml/org/kde/desktop/private/TextFieldContextMenu.qml" and "qrc:/qt/qml/org/kde/desktop/MenuItem.qml"
```

This brings up the touchpad settings pane, with all settings showing

As seen by the output, kcm_touchpad has different errors on the two systems. On the Dell, it looks to me like the error from the back end is the issue

The system does detect the touchpad

```
~ ❯❯❯ xinput list
⎡ Virtual core pointer                          id=2    [master pointer  (3)]
⎜   ↳ Virtual core XTEST pointer                id=4    [slave  pointer  (2)]
⎜   ↳ ZSA Technology Labs ErgoDox EZ Glow       id=15   [slave  pointer  (2)]
⎜   ↳ ZSA Technology Labs ErgoDox EZ Glow Consumer Control      id=13   [slave  pointer  (2)]
⎜   ↳ Logitech G502                             id=16   [slave  pointer  (2)]
⎜   ↳ DELL0A5D:00 04F3:311C Mouse               id=17   [slave  pointer  (2)]
⎜   ↳ DELL0A5D:00 04F3:311C Touchpad            id=18   [slave  pointer  (2)]
⎜   ↳ PS/2 Generic Mouse                        id=25   [slave  pointer  (2)
```

**Comment by TraceyC77** [comment]
I found an [upstream issue](https://bugs.kde.org/show_bug.cgi?id=483812) which also has the solution. The bug I experienced may be due to [Plasma dropping synaptics config](https://invent.kde.org/plasma/plasma-desktop/-/commit/12ac086128c7e7db831f35d53616db0d6823fd90)

To reiterate the soluion here:

I had xorg-driver-input-synaptics installed because in Plasma 5, it was necessary to be able to see all options for my touchpad. (Otherwise, some checkboxes were not present)

After uninstalling xorg-driver-input-synaptics and rebooting, I now see all settings in the Touchpad settings. Closing this issue.
